client/signup/steps/email-signup-form/index.jsx

```diff
@@ -38,6 +39,11 @@ export default React.createClass( {
 
 		analytics.tracks.recordEvent( 'calypso_signup_user_step_submit', analyticsData );
 
+		// run and record before creating user
+		if ( ! this.props.signupDependencies.cartItem ) {
+			abtest( 'nuxTrampoline' );
```

Comment:
This will assign any user that chose the free plan, including those in the existing DSS and vertical survey tests. We can use `page` to check the current URL isn't in those flows, and/or `getABTestVariation` again to make sure we're not stepping on other tests.